architecture/juce/juce-plugin.cpp

```diff
@@ -237,6 +242,9 @@ class FaustPlugInAudioProcessor : public foleys::MagicProcessor, private juce::T
     void processBlock (juce::AudioBuffer<float>& buffer, juce::MidiBuffer& midiMessages) override
     {
         jassert (! isUsingDoublePrecision());
+#ifdef MAGIC_LEVEL_SOURCE
+	outputMeter->pushSamples(buffer);
```

Comment:
If you push the samples before the `process (buffer, midiMessages)` it is actually an inputMeter ;-)
You could have both though if you want.

Reply:
Will review.

Reply:
Somehow I had deleted process (buffer, midiMessages) from this completely while I was working on the code.  It's pretty difficult to tell whether the metering is on input of output.  Where is there further documentation on this?

Reply:
If you push the samples to the meter before processing it's input, if you push after processing, it's output. ;-)
If you click on "View changes" you will see the side-by-side diff, which shows a bit more context.

Reply:
Ah, I guess it makes sense.  The buffer is changed after the process() call.